docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -84,14 +81,13 @@ To save the `logs@lifecycle` policy as a new policy in {kib}:
 . On the **Edit policy logs** page, toggle **Save as new policy**, and then provide a new name for the policy, for example, `logs-custom`.
 
 The `logs@lifecycle` policy uses the recommended rollover defaults: Start writing to a new
-index when the current write index reaches 50GB or becomes 30 days old.
+index when the primary shard reaches 50GB or the index becomes 30 days old.
 
 To view or change the rollover settings, click **Advanced settings** for the hot
-phase. Then disable **Use recommended defaults** to display the rollover
-settings.
+phase. Adjust the **Minimum primary shard size** and other rollover settings as needed.
 
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]
+image::images/ilm/tutorial-ilm-hotphaserollover-new.png[View rollover defaults]
```

Comment:
This image doesn't exist, do you need to add it to git so it's committed for this PR?